plugins/net_plugin/net_plugin.cpp

```diff
@@ -412,10 +414,11 @@ namespace eosio {
     *  the need for compatibility hooks
     */
    constexpr uint16_t proto_base = 0;
-   constexpr uint16_t proto_explicit_sync = 1;
-   constexpr uint16_t block_id_notify = 2; // reserved. feature was removed. next net_version should be 3
+   constexpr uint16_t proto_explicit_sync = 1; // version at time of eosio 1.0
+   constexpr uint16_t block_id_notify = 2;     // reserved. feature was removed. next net_version should be 3
+   constexpr uint16_t pruned_types = 3;        // support packed_transaction & signed_block in addition to prior *_v0 versions
 
-   constexpr uint16_t net_version = proto_explicit_sync;
+   constexpr uint16_t net_version = pruned_types;
```

Comment:
I think we should hold off on supporting the new network protocol version in this PR. This PR should just make the changes in net_plugin necessary to continue supporting the existing protocols given that it now needs to convert between the legacy and new types.

Reply:
Reverted.